src/ext/components/controls.coffee

```diff
@@ -0,0 +1,559 @@
+{ defer, filter, find, flatten, forEach, groupBy, map, throttle, uniqueId } = require('lodash')
```

Comment:
@honzasterba 
this lists all the generic controls used by `ModelBuilder` (they were mainly in `model-input.coffee` before)  and `AutoML` + the more abstract controls that were in `ext/modules/util` (for some strange reason`.
My changes there:
- some fixes to `createTextboxControl`: fixed the value retrieval when value is a number ()was working only for arrays) + when it's empty string (returns null)
- some improvements to `createListControl`: making navigation buttons and columns message deactivable.
- extracted control groups creation logic + creation of controls from parameters to object `ControlGroups` which is the controls entry point used by `model-input.coffee` and `automl-input.coffee`.

Reply:
Reason for extraction is simply because it was in `model-input` component before.
Having it separated should make us aware that those components are not only used by model builder only.